server/service/src/sync/program_schemas/hiv_care_program_ui_schema.json

```diff
@@ -10,7 +10,8 @@
       "type": "Control",
       "scope": "#/properties/enrolmentDatetime",
       "options": {
-        "readonly": true
+        "readonly": true,
+        "dateOnly": true
```

Comment:
As requested in https://github.com/openmsupply/open-msupply/issues/644

Reply:
Could you please at a label here because it shows datetime...

Also the readonly flag is not working for the calendar button and you can still select a different date...

There is also the HIV testing program that would need a similar change

Reply:
Yup, I actually did this in #839, but I've merged that in here now so you should see it. Also done in the other one too: b2aaee0f2